src/DotNet.Status.Web/Controllers/GitHubHookController.cs

```diff
@@ -251,15 +250,11 @@ private async Task ProcessRcaRulesAsync(IssuesHookData data, string action)
                 assignee = data.Sender.Login;
             }
 
-            _logger.LogInformation("Opening RCA work item in Azure Boards {org}/{project}", _azureDevOpsOptions.Value.Organization, _azureDevOpsOptions.Value.Project);
+            _logger.LogInformation("Opening RCA work item in Azure Boards {org}/{project}", _rcaOptions.Value.Organization, _rcaOptions.Value.Project);
```

Comment:
I think you took away the rcaOptions.Value.Project from the settings file. Is this correct?